.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: ruby
-sudo: false
```

Comment:
Removing this because it's now default, or is it related to this PR?

Reply:
it isn't necessary to fix tests but it's recommended to remove the line, see
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration